docs/content/users/install/docker-installation.md

```diff
@@ -28,7 +28,7 @@ You’ll need a Docker provider on your system before you can [install DDEV](dde
     When your computer restarts, you’ll need to `colima start` again. This will eventually be automated in later versions of Colima.
     
     !!!tip "Colima disk allocation"
-        We recommend allocating lots of storage for Colima because there’s no way to increase the size later. You can reduce usage with `ddev clean`, and kill off disk images with `docker rm -f $(docker ps -aq) && docker rmi -f $(docker images -q)`. If you have to rebuild your Colima instance, use the technique described below for migrating from Docker Desktop.
+        We recommend allocating lots of storage for Colima because there’s no easy way to increase the size later (see [github comment](https://github.com/abiosoft/colima/issues/398#issuecomment-1449801811) for a step-by-step guide to resize disk images with `qemu`). You can reduce usage with `ddev clean`, and kill off disk images with `docker rm -f $(docker ps -aq) && docker rmi -f $(docker images -q)`. If you have to rebuild your Colima instance, use the technique described below for migrating from Docker Desktop.
```

Comment:
I’d capitalize the proper noun and “run” commands:

```suggestion
        We recommend allocating lots of storage for Colima because there’s no easy way to increase the size later (see [GitHub comment](https://github.com/abiosoft/colima/issues/398#issuecomment-1449801811) for a step-by-step guide to resize disk images with `qemu`). You can reduce usage by running `ddev clean`, and killing off disk images by running `docker rm -f $(docker ps -aq) && docker rmi -f $(docker images -q)`. If you have to rebuild your Colima instance, use the technique described below for migrating from Docker Desktop.
```